### DISCUSSION TITLE
Guardian Dashboard

### DISCUSSION THREAD

**Discussion by garyray-k** [fedimint/fedimint#2475] [General]
What sort of information do you want to see on the Guardian dashboard?

1. BTC amount
2. System stats (CPU/disk space/etc)

**Comment by sahilc0** [comment]
- Invite to federation / QR
- if no lightning gateway recognized, maybe a reminder/link to set that up
https://www.figma.com/file/ioeAVbWBc3ow2RaZETatM7/Fedimint-Setup-Flow?type=design&node-id=27-2456&t=a3NjBa7zY3hdWWs4-11

**Comment by m1sterc001guy** [comment]
Some random thoughts from stuff I usually just manually query from the database
 - uptime
 - A tab per module with module specific data
 - Module code version/module database version (might want a tab or something with module specific metadata)
 - Federation peers and connection status (maybe a name per peer?)
 - Peer status (i.e disconnected, banned, online, etc)
 - Maybe an "epoch" explorer? Not sure if this would leak some privacy though and open some side channel attacks. Perhaps with epochs going away with the new consensus algorithm this might not be helpful  (@jkitman might have some thoughts here).
 - Number of AcceptedTransactions
 - RejectedTransactions and reason

**Reply by jkitman** [reply]
I would avoid anything "epoch" related.

**Reply by m1sterc001guy** [reply]
Perhaps I'm confused. Is there a difference between the fed admin UI and the guardian admin UI?

**Comment by jkitman** [comment]
List of known federations (including perhaps ones we haven't registered with yet).  We should display the following information for each fed (as well as an aggregate number up top):

- Custodial funds
- Number of LN payments sent/received on behalf of users
- Amount of revenue earned
- Current fee rate
- Ideally we could display aggregate numbers on these (self-reported, but verifiable) by the fed

Actions we could take with each fed:

- Register with fed (perhaps this can also update our routing hints, fees, and other info)
- Withdraw/deposit funds on-chain

**Reply by m1sterc001guy** [reply]
Would these be for the guardian UI or for the gateway UI?

**Reply by jkitman** [reply]
Whoops thought this was about the gateway since that's what we talked about in the design discussion :sweat_smile:

**Comment by yrp237** [comment]
Number of Fedi members total,
Average length of membership for all members,
Newest members tab (as guardians may want to reach out & say hello)

These may make sense for both guardians & members but for sure guardians.

**Reply by justinmoon** [reply]
Unfortunately, none of these are possible. Fedimint doesn't have "users". User actions aren't linked to any shared identity.
